pom.xml

```diff
@@ -148,7 +148,7 @@
     <!-- required by zookeeper test utilities imported from ZooKeeper -->
     <junit5.version>5.8.2</junit5.version>
     <libthrift.version>0.14.2</libthrift.version>
-    <lombok.version>1.18.22</lombok.version>
+    <lombok.version>1.18.24</lombok.version>
```

Comment:
please upgrade dependencies in a separate PR

Reply:
@eolivelli reverted